pkg/services/ngalert/api/api_alertmanager_test.go

```diff
@@ -357,7 +357,8 @@ func createSut(t *testing.T, accessControl accesscontrol.AccessControl) Alertman
 		accessControl = acMock.New().WithDisabled()
 	}
 	log := log.NewNopLogger()
-	return AlertmanagerSrv{mam: mam, store: &configStore, secrets: secrets, ac: accessControl, log: log}
+	crypto := notifier.NewCrypto(secrets, &configStore, log)
+	return AlertmanagerSrv{mam: mam, crypto: crypto, ac: accessControl, log: log}
```

Comment:
```suggestion
	return AlertmanagerSrv{mam: mam, crypto: mam.Crypto, ac: accessControl, log: log}
```

Reply:
Committed this on next branch to ensure it won't be lost.